views/js/component/results/list.js

```diff
@@ -57,8 +57,8 @@ define([
         function pluginRun(method) {
             var execStack = [];
 
-            _.forEach(plugins, function (plugin) {
-                if (_.isFunction(plugin[method])) {
+            plugins.forEach(function (plugin) {
```

Comment:
`plugins` is an object.

```suggestion
            Object.entries(plugins).forEach(function ([pluginName, plugin]) {
```

Or switch to `for ... in` loop for simplicity/